tools/analyzer_plugin/lib/src/diagnostic/missing_required_prop.dart

```diff
@@ -221,6 +236,20 @@ class MissingRequiredPropDiagnostic extends ComponentUsageDiagnosticContributor
       }
     }
 
+    if (forwardedProps != null) {
+      debugHelper.log(() {
+        var message = StringBuffer()..writeln(forwardedProps);
+        if (debugSuppressedRequiredPropsDueToForwarding.isNotEmpty) {
+          final propsNamesByClassName = <String?, Set<String>>{};
```

Comment:
Is a null key here indicative of things like ubiquitive / un-namespaced props - or something else?

Reply:
No, it's just that technically `.name` is nullable here, since `field.enclosingElement` is typed as `Element` and not `InterfaceElement` (which has a non-nullable `.name`).

So in practice, this should never be null